src/components/common/Button/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { TouchableOpacity, Text } from 'react-native';
+import styles from './styles';
+
+const Button = ({ title, testId, ...props }) => {
+  return (
+    <TouchableOpacity style={styles.button} testID={testId} {...props}>
+      <Text style={styles.buttonTitle}>{title}</Text>
+    </TouchableOpacity>
+  );
+};
+
+Button.propTypes = {
+  title: string.isRequired,
+  testId: string,
```

Comment:
WDYT about doing
```suggestion
  testID: string,
```
for naming consistency